peer/network.go

```diff
@@ -100,12 +103,12 @@ func NewNetwork(appSender common.AppSender, codec codec.Manager, self ids.NodeID
 	}
 }
 
-// RequestAny synchronously sends request to a randomly chosen peer with a
+// SendRequestAny synchronously sends request to an arbitrary peer with a
 // node version greater than or equal to minVersion. If minVersion is nil,
 // the request will be sent to any peer regardless of their version.
 // Returns the ID of the chosen peer, and an error if the request could not
 // be sent to a peer with the desired [minVersion].
-func (n *network) RequestAny(minVersion *version.Application, request []byte, handler message.ResponseHandler) (ids.NodeID, error) {
+func (n *network) SendRequestAny(minVersion *version.Application, request []byte, handler message.ResponseHandler) (ids.NodeID, error) {
```

Comment:
can we rename those to `SendAppRequestAny`, for a second I thought this was also being used for Crosschains as well.

Reply:
yup